packages/dataclass-editor/src/components/dataclass/detail/DataClassDetailContent.tsx

```diff
@@ -32,34 +42,46 @@ export const DataClassDetailContent = () => {
 
   return (
     <Flex direction='column' gap={4} className='detail-content'>
-      <BasicField label='Class type'>
-        <ToggleGroup type='single' className='class-type-group' value={initialClassType} onValueChange={handleClassTypeChange}>
-          <ToggleGroupItem value='DATA' asChild>
-            <Button variant={variant('DATA')} size='large'>
-              Data
-            </Button>
-          </ToggleGroupItem>
-          <ToggleGroupItem value='BUSINESS_DATA' asChild>
-            <Button variant={variant('BUSINESS_DATA')} size='large'>
-              Business Data
-            </Button>
-          </ToggleGroupItem>
-          <ToggleGroupItem value='ENTITY' asChild>
-            <Button variant={variant('ENTITY')} size='large'>
-              Entity
-            </Button>
-          </ToggleGroupItem>
-        </ToggleGroup>
-      </BasicField>
       <BasicField label='Description'>
         <Textarea value={dataClass.comment} onChange={event => handleDataClassPropertyChange('comment', event.target.value)} />
       </BasicField>
```

Comment:
to hold it constistant I would also wrap this into a collapsible
And why not also add the dataclass name to it? (of course disabled)

Reply:
The name can be seen in the header already, I don't think it's necessary to add it.

Reply:
If you understand that this is the name, your right. But maybe for some people this info could be helpful.
But was only an idea ;)